docs/src/app/components/pages/customization/using-context.md

```diff
@@ -0,0 +1,135 @@
+## Using Context
+
+The `MuiThemeProvider` component simply adds the `muiTheme` object and `uniqueIdGen` function to context.
+If you prefer using context directly instead, you can follow this pattern:
+
+### Pass Theme
+
+Pass theme down the context:
+
+```js
+import React from 'react';
```

Comment:
We moved to using the following pattern for our code source. Shouldn't we advice user to do the same?
```js
import React, {Component} from 'react';
```